migrations/educandu-2022-07-18-02-migrate-document-models.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable camelcase */
+
+export default class Educandu_2022_07_18_02_migrate_documents_models {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    const props = {
+      roomId: '',
```

Comment:
I would opt for `null` if intentionally if no room is assigned. I know that the `restoredFrom` field also is an empty string, but I'd consider that an "accident". In the sections e.g. `deletedBy` (which also is not an arbitrary string, but an ID reference) is `null` if not applicable (so are all date types). I would interpret empty string as "text intentionally left empty by the user" opposed to `null` (data value absent as part of the domain model). Of course, that's disputable. What do you think?

Reply:
I think we should keep `null` (as we do for all other user data), as a lack of data, not as an intentionally set empty data by the user.